Makefile

```diff
@@ -33,10 +33,11 @@ KEPLER_VERSION_LIBBPF ?=release-0.6.1-libbpf
 # sustainable_computing_io/kepler-operator-bundle:v$VERSION and sustainable_computing_io/kepler-operator-catalog:$VERSION
 IMG_BASE ?= quay.io/sustainable_computing_io
 
+KEPLER_OPERATOR_IMG_BASE :=$(IMG_BASE)
```

Comment:
sorry, if I wasn't clear last time. We want to `IMG_BASE` for operator image. and `KEPLER_IMG_BASE` for all **kepler** related images. 👼 .